### PR TITLE
[Trivial] Add new member & update CODEOWNERS

### PR DESCRIPTION
**Pull request by DonghakPark**
Add new member & update CODEOWNERS

**Changes proposed in this PR:**
    modified:   .github/CODEOWNERS
    modified:   CONTRIBUTING.md
    modified:   README.md

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Review by djeong20 — APPROVED** [review]
lgtm

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -7,4 +7,4 @@
 # all member IDs directly to avoid unexpected situations.
 
 # In order that all members of a repository are supposed to review each other
-* @nnstreamer/nnstreamer @myungjoo @jijoongmoon @again4you @jaeyun-jung @leemgs @wooksong @helloahn @kparichay @dongju-chae @gichan-jang @anyj0527 @zhoonit
+* @nnstreamer/nnstreamer @myungjoo @jijoongmoon @again4you @jaeyun-jung @leemgs @wooksong @helloahn @kparichay @dongju-chae @gichan-jang @anyj0527 @zhoonit @lhs8928 @songgot @jihochu @DonghakPark @SeoHyungjun @baek2sm @skykongkong8 @djeong20 @EunjuYang
```
[the remark this answers is not in the document]
Comment (myungjoo) [reply .github/CODEOWNERS:10]: She ignored the previous invitation, which is now expired. Tell her to accept the invitation. :) @EunjuYang

[the remark this answers is not in the document]
Comment (EunjuYang) [reply .github/CODEOWNERS:10]: Oh, I missed the previous invitation! I check the new one and accept it. Thank you :)

**Review by jihochu — APPROVED** [review]
LGTM

**Review by lhs8928 — APPROVED** [review]
LGTM

**Pull request merged**